### PR TITLE
Adds checks with http.uri keyword and HTTP2 traffic

### PR DESCRIPTION
**Pull request by catenacyber**

### CODE REVIEW

File: tests/http2-basic/test.rules [hunk 1]
```diff
@@ -2,3 +2,8 @@ alert http2 any any -> any any (http2.header; content:"agent: nghttp2"; sid:1; r
 alert http2 any any -> any any (http2.frametype:GOAWAY; sid:2; rev:1;)
 alert http2 any any -> any any (http2.settings:SETTINGS_HEADER_TABLE_SIZE>1000; sid:3; rev:1;)
 alert http2 any any -> any any (http2.window:34634; sid:4; rev:1;)
+
+alert http any any -> any any (http.uri; content:"/doc/manual/html/index.html"; sid:10;)
+alert http1 any any -> any any (http.uri; content:"/doc/manual/html/index.html"; sid:11;)
+alert http2 any any -> any any (http.uri; content:"/doc/manual/html/index.html"; sid:12;)
+alert ip any any -> any any (http.uri; content:"/doc/manual/html/index.html"; sid:13;)
```
Comment (victorjulien) [inline tests/http2-basic/test.rules:9]: can you also add a `alert tcp` variant here?

**Comment by victorjulien** [comment]
I think it would be good to test `alert http1` in a HTTP/1 test as well.

File: tests/http2-basic/test.rules (hunk 1 above)
Comment (catenacyber) [reply tests/http2-basic/test.rules:9]: Sure. Why should it be different than `ip` ?

**Comment by catenacyber** [comment]
Replaced by https://github.com/OISF/suricata-verify/pull/382

**Pull request closed without merging**